src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromDevider = fromFormat[3];
+  const toDevider = toFormat[3];
+
+  const dateValues = date.split(fromDevider);
+
+  const objectDate = {};
+
+  fromFormat.forEach((key, value) => {
```

Comment:
The parameters for the callback function in 'forEach' are in the wrong order. It should be '(value, index)' instead of '(key, value)'.